### PR TITLE
Clean up

### PR DESCRIPTION
**Pull request by assertchris**
Some long-overdue housekeeping...

### CODE REVIEW

**Comment by assertchris** [comment]
Sorry to do this to you, @tractorcow. :/

File: code/compat/pages/BlogEntry.php [hunk 1]
```diff
@@ -2,59 +2,82 @@
 
 /**
  * @deprecated since version 2.0
+ *
+ * @property int ParentID
```
Comment (tractorcow) [inline code/compat/pages/BlogEntry.php]: Can we use `$` as in `@property int $PropertyID`? See http://www.phpdoc.org/docs/latest/references/phpdoc/tags/property.html

File: code/compat/pages/BlogTree.php [hunk 2]
```diff
@@ -4,19 +4,29 @@
...
+	/**
+	 * {@inheritdoc}
+	 */
 	public function canCreate($member = null) {
-		// Deprecated
 		return false;
 	}
 
+	/**
+	 * {@inheritdoc}
```
Comment (tractorcow) [inline code/compat/pages/BlogTree.php]: `{@inheritcomment}`

File: code/compat/tasks/BlogMigrationTask.php [hunk 3]
```diff
@@ -1,34 +1,24 @@
...
- */
 class BlogMigrationTask extends MigrationTask {
-
 	/**
 	 * Should this task be invoked automatically via dev/build?
 	 *
 	 * @config
-	 * @var boolean
+	 *
+	 * @var bool
```
Comment (tractorcow) [inline code/compat/tasks/BlogMigrationTask.php]: Stop boolying my code :D

File: code/compat/tasks/BlogMigrationTask.php [hunk 4]
```diff
@@ -37,40 +27,66 @@ public function up() {
...
 			} else {
-				// Migrate object
 				$this->upClass($class);
 			}
 		}
 	}
 
-	public function down() {
-		$this->message('BlogMigrationTask::down() not implemented');
+	/**
```
Comment (tractorcow) [inline code/compat/tasks/BlogMigrationTask.php]: What method ordering convention are you using?

File: code/compat/tasks/BlogMigrationTask.php (hunk 3 above)
Comment (assertchris) [inline code/compat/tasks/BlogMigrationTask.php]: Very good. :)

File: code/compat/tasks/BlogMigrationTask.php (hunk 4 above)
Comment (assertchris) [inline code/compat/tasks/BlogMigrationTask.php]: Dependant methods together, depth-first (as recommended in http://www.amazon.com/dp/0132350882).

**Pull request merged**